lib/scheduler.js

```diff
@@ -183,4 +183,12 @@ Scheduler.prototype.stopServer = function(successCallback, failureCallback) {
   }
 }
 
+
+/**
+ * Get scheduler status
+*/
+Scheduler.prototype.isRunning = function() {
```

Comment:
recent node version has server.listening property

https://nodejs.org/api/http.html#http_server_listening
node/lib/net.js in https://github.com/nodejs/node 
```javascript
Object.defineProperty(Server.prototype, 'listening', {
  get: function() {
    return !!this._handle;
  },
  configurable: true,
  enumerable: true
});
```

so, it's better to check this.server !== null & this.server._handle !== null